src/python/pants/notes/master.rst

```diff
@@ -4,7 +4,12 @@ Master Pre-Releases
 This document describes ``dev`` releases which occur weekly from master, and which do
 not undergo the vetting associated with ``stable`` releases.
 
-1.9.0rc0 (07/18/2018)
+1.9.0rc1 (07/20/2018)
```

Comment:
This won't be release from master, so no edits needed in `master.rst`.

Reply:
Removed the `1.9.0rc1` entry from `master.rst` in d3b5d9d!

Reply:
Did we change the process recently? iirc, the notes still needs to be on master.

Reply:
oh nvm. `master.rst` doesn't need to change.